EIPS/eip-7198.md

```diff
@@ -0,0 +1,33 @@
+---
+eip: 7198
```

Comment:
```suggestion
eip: 7199
```
Assigning the EIP number as PR number as standard practice rather than using the issue number #7198.  
I recommend following standard practice for EIP editors so there isn't any perception of special treatment.